### ISSUE TITLE
Error invalid syntax poolFreqDiffTest_QBGLM.py

### ISSUE THREAD

**Issue by msalamon2** [RAWWiberg/poolFreqDiff#1]
Hi, 

I am trying to run the poolFreqDiff analysis on a pool sync file and got the following error message:

File "~/programs/poolFreqDiff/poolFreqDiffTest_QBGLM.py", line 117
    exec "sample_%s=pop[sam]" %(sam)
            ^
SyntaxError: invalid syntax

I also got error message from an indentation issue on line 98 and was able to fix it.

I am not sure how to fix the syntax error and would appreciate your help.

Thanks
Mathilde

**Comment by RAWWiberg** [comment]
@msalamon2 Thanks for leaving this up. It will be good for future reference.
We determined that this type of error is due to the python version. I wrote these scripts for `python2.7`, `python3` requires different syntax for various functions (e.g. `print()` instead of `print` , and `exec()` instead of `exec`).

I will try to make new scripts that will also work for `python3`, to keep things up-to-date.
